src/qt/bitcoin.cpp

```diff
@@ -466,6 +468,10 @@ int GuiMain(int argc, char* argv[])
     QApplication::setOrganizationName(QAPP_ORG_NAME);
     QApplication::setOrganizationDomain(QAPP_ORG_DOMAIN);
     QApplication::setApplicationName(QAPP_APP_NAME_DEFAULT);
+    if (gArgs.IsArgSet("-guisettings") && gArgs.GetArg("-guisettings", "") != "") {
```

Comment:
Is `gArgs.IsArgSet("-guisettings")` needed here?

Reply:
You're right, forgot that